package.json

```diff
@@ -55,15 +47,25 @@
     "@types/hosted-git-info": "^3.0.2",
     "@types/lru-cache": "^5.1.1",
     "@types/node": "^16.11.9",
+    "@types/react": "^17.0.35",
     "@types/styled-components": "^5.1.15",
+    "@typescript-eslint/eslint-plugin": "^5.4.0",
+    "@typescript-eslint/parser": "^5.4.0",
     "eslint": "^8.2.0",
+    "eslint-plugin-import": "^2.25.3",
+    "eslint-plugin-jest": "^25.2.4",
+    "eslint-plugin-json": "^3.1.0",
+    "eslint-plugin-react": "^7.27.1",
+    "eslint-plugin-react-hooks": "^4.3.0",
+    "eslint-plugin-simple-import-sort": "^5.0.3",
+    "eslint-plugin-testing-library": "^5.0.0",
     "husky": "^7.0.4",
     "lint-staged": "^12.0.3",
     "markdownlint-cli": "^0.30.0",
     "ncp": "^2.0.0",
     "npm-run-all": "^4.1.5",
     "prettier": "~2.2.1",
-    "typescript": "^4.4.4"
+    "typescript": "4.4.4"
```

Comment:
https://github.com/microsoft/TypeScript/issues/46874

Reply:
See https://github.com/yarnpkg/berry/issues/3722#issuecomment-974634888

Reply:
Nice, thanks! Running `yarn set version canary && yarn install` allowed me to use TS 4.5.2 with PnP 🎉

I’m now getting a bunch of `--experimental-loader` warnings, but these can be looked into separately.

<img width="1007" alt="Screenshot 2021-11-21 at 21 21 10" src="https://user-images.githubusercontent.com/608862/142779378-9b7cbb78-d5a5-4caf-bcab-871f5e8bf01d.png">

Reply:
Asked about the warnings here: https://github.com/yarnpkg/berry/issues/3722#issuecomment-974900819

Reply:
Answered in https://github.com/yarnpkg/berry/issues/3722#issuecomment-974908156